### DISCUSSION TITLE
No way to separate dependencies upon install?

### DISCUSSION THREAD

**Discussion by ralphieb2t** [wstyres/Zebra#1948] [Q&A] [answered]
Not really sure how to word this but, is there no way to separate linked packages upon install? Let me just give an example; I have Tweak X currently installed- which is on multiple repositories, if I go to install another Tweak that requires Tweak X from a diff repo, how can I separate? Or mark that “Tweak X”  has already been installed through another repo??  
p.s sorry if I should have put this in the new feature section, I just wasn’t sure if there’s already a way to do it and I’m just overlooking something. Thanks.

**Comment by kirb** [comment, marked as answer]
In general in a situation like this it seems like you’d already be aware that you need a specific version of a dep, so you should install the right version manually first, or over the top afterwards. It would be tricky to come up with UI for this in the queue screen that takes UX into account - i.e. most users should be using latest versions, we don’t want to make it easy to unintentionally downgrade and experience bugs that were already fixed, or install from a repo with a lower priority (Elucubratus/Procursus repos are pinned so other repos can’t override critical packages). Or am I misunderstanding what you’re suggesting?

**Reply by ralphieb2t** [reply]
Yeah, I think that’s the issue. That makes a lot of sense, I appreciate the reply— even if it’s just working through the thought. Anyway, After reading what you said & re-examining, I believe that’s what happened in this scenario; to be specific(hopefully no one minds, credit to them) I had “AppSync Unified” somehow installed from CokerNutx’s repo, since I guess they’ve got a mirror or it was avail there prior, then when I went to download an .ipa thru Zbra that needed signing, I guess the Dev’s thinking was to use AppSync for convenience. 

Since, I’m guessing, I didn’t have it installed thru Karen’s Repo, even though technically I had the Tweak already installed, Zbra linked them together wanting me to install the Karen repo one, fortunately I have a good memory for the tweaks I install otherwise I’m afraid what that might’ve done(instlling Appsync over Appsync)— I mean, unless it’s smart enough to know, *you already have this tweak installed* that would be pretty cool too! …but yeah, after looking it over, that’s abt how it went down lol honestly, it did me a favor because I think the one from CokerNutx might be an old ver. as is. So you’re more clever than I am haha. Thanks.
